release-notes/createReleaseNotes.sh

```diff
@@ -0,0 +1,8 @@
+branch=`git rev-parse --abbrev-ref HEAD`
+file=release-notes/unreleased/$branch.yml
+echo $branch
+prefix=${branch:0:7}
```

Comment:
Probably won't work for issue numbers < 100
You could use something like the following though:
```
awk '{ match($0, /LUN-[0-9]+/, arr); print arr[0] }'
```